src/pgrn-custom-scan.c

```diff
@@ -764,6 +767,97 @@ PGrnBeginCustomScan(CustomScanState *customScanState,
 	memset(&(state->searchData), 0, sizeof(state->searchData));
 }
 
+static TupleTableSlot *
+PGrnResultTupleSlot(CustomScanState *customScanState,
```

Comment:
Could you use verb for function name?
`PGrnMakeTupleTableSlot()`? `PGrnBuildTupleTableSlot()`?